system/kube-monitoring/charts/prometheus-frontend/openstack-jump.alerts

```diff
@@ -47,3 +47,15 @@ groups:
     annotations:
       description: 'Datapath {{ $labels.service }} {{ $labels.check }} is flapping for 30 minutes. See Sentry for details'
       summary: 'Datapath {{ $labels.service }} {{ $labels.check }} is flapping'
+
+  - alert: JumpserverRSAForwardingDown
+    expr: jump_rsa_forwarding_up == 1 or absent(jump_rsa_forwarding_up)
+    for: 15m
+    labels:
+      context: availability
+      service: jumpserver
+      severity: warning
```

Comment:
Suggesting to start with `info` level to test and tune the alert for some days. Feel free to adjust the severity to `warning` after testing.

Reply:
Good idea.